packages/flutter/lib/src/material/popup_menu.dart

```diff
@@ -264,15 +264,18 @@ class PopupMenuItem<T> extends PopupMenuEntry<T> {
   /// of [ThemeData.textTheme] is used.
   final TextStyle? textStyle;
 
+  /// {@template flutter.material.popupmenu.mouseCursor}
   /// The cursor for a mouse pointer when it enters or is hovering over the
   /// widget.
   ///
   /// If [mouseCursor] is a [MaterialStateProperty<MouseCursor>],
   /// [MaterialStateProperty.resolve] is used for the following [MaterialState]:
```

Comment:
Since there's only one state, it's OK to be a little less formal:
`If [mouseCursor] is a [MaterialStateProperty<MouseCursor>], [MaterialStateProperty.resolve] [MaterialState.disabled].`